tests/testthat/_snaps/query-select.md

```diff
@@ -13,6 +13,6 @@
     Code
       lf_render
     Output
-      <SQL> SELECT `x`
+      <SQL> SELECT *
```

Comment:
This comes from a minor optimisation that was necessary to produce mostly the same SQL as before